### DISCUSSION TITLE
fix the arguments used in inpainting_inferencer 修复 inpainting_inferencer 的传参问题

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#590] [MMagic]
# fix the arguments used in inpainting_inferencer 
 修复 inpainting_inferencer 的传参问题 

please find more details about the problem and the provided solution in [https://github.com/open-mmlab/mmagic/issues/1945](https://github.com/open-mmlab/mmagic/issues/1945) 

问题描述和解决方案详见 [https://github.com/open-mmlab/mmagic/issues/1945](https://github.com/open-mmlab/mmagic/issues/1945) 

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：None

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by ShaoqLin** [comment]
Claim the mission
